vcontrol/rest/providers/providers.txt

```diff
@@ -0,0 +1 @@
+TEST : TEST
```

Comment:
I'd rather not have this included in the repo, but instead have the test create this file for testing, and then clean up after itself after the test runs.